addons/l10n_pe_website_sale/demo/website_demo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+
+    <record id="default_website_pe" model="website">
+        <field name="name">(PE) Peruvian Website</field>
+        <field name="company_id" ref="l10n_pe.demo_company_pe" />
+        <field name="logo" type="base64" file="l10n_pe_website_sale/static/description/icon.png" />
+        <field
+            name="domain"
+            model="ir.config_parameter"
+            eval="obj().env['ir.config_parameter'].sudo().get_param('web.base.url')"
+        />
+    </record>
+    <function model="payment.provider" name="copy">
+        <value eval="[ref('payment.payment_provider_transfer')]" />
+        <value eval="{'company_id': ref('l10n_pe.demo_company_pe'), 'state': 'enabled'}" />
+    </function>
+    <function model="product.pricelist" name="write">
+        <value
+            model="product.pricelist"
+            eval="obj().search([('currency_id', '=', ref('base.PEN')), ('company_id', '=', ref('l10n_pe.demo_company_pe'))]).id"
+        />
+        <value model="website" eval="{'sequence': 1, 'website_id': ref('l10n_pe_website_sale.default_website_pe')}" />
+    </function>
+    <function model="product.product" name="write">
```

Comment:
I don't think you need this to use the website

Reply:
What is expected with this is that when an order is created, the billing does not fail.

Since in Peru, it is mandatory to have a valid tax.

We can remove it if you want.

cc @AndreaGeraldo

Reply:
seems really overkill to assign taxes to every demo products